README_CN.md

```diff
@@ -295,35 +295,6 @@ ESP32_Display_Panel 会根据 [ESP_Panel_Board_Custom.h](./ESP_Panel_Board_Custo
 
 ## 其他相关说明
 
-### 配置支持的开发板
-
-下面是在不同开发板上开发 GUI 应用程序的建议配置，这些设置能够根据实际需求进行调整，用户可以前往 Arduino IDE 中的工具菜单来配置以下设置
```

Comment:
Please leave this part and add the messages like below:

```
关于如何在 Arduino IDE 中配置支持的开发板，请参考 [说明](src/board/Board_Instructions.md#recommended-configurations-in-the-arduino-ide).
```

Reply:
done